src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -122,8 +122,7 @@ public async Task UpdateFileStorageAsync()
                 };
                 roslynSettingsFileStorage.Update(settings);
             }
-
-            logger.LogDebugExtended("End");
+            CodeMarkers.Instance.FileSynchronizerUpdateStop();
```

Comment:
Just to point out that this won't be logged if we have an exception in the middle. I don't know if it's relevant from ETW's perspective to have the Stop when there are errors.

Reply:
I don't think ETW cares - it just logs whatever events it is given. I'm guessing that the matching of Start/Stop event pairs is done by PerfView.

We have try-finally blocks for the other event pairs, although for those the try-catch already existed. However, we may as well be consistent and have a try-finally so we always have matching pairs of events - it will make the logs easier to understand.